src/ucp/rma/rma.inl

```diff
@@ -129,4 +131,21 @@ ucp_rma_sw_do_am_bcopy(ucp_request_t *req, uint8_t id, ucp_lane_index_t lane,
     return (ucs_status_t)packed_len;
 }
 
+static UCS_F_ALWAYS_INLINE ucs_status_t
+ucp_proto_request_rma_zcopy_init(ucp_request_t *req, ucp_md_map_t md_map,
```

Comment:
maybe call it ucp_proto_request_multi_zcopy_init - since it's not specific to rma - and move it to port_multi.inl, and reuse it from ucp_proto_request_eager_zcopy_multi_init